index.qmd

```diff
@@ -41,19 +41,17 @@ remotes::install_github("topepo/ongoal@hockeyR")
 
 These slides are designed to use with live teaching and are published for workshop participants' convenience. There are not meant as standalone learning materials. For that, we recommend [tidymodels.org](https://www.tidymodels.org/start/) and [*Tidy Modeling with R*](https://www.tmwr.org/).
 
-### Day One
+### Introduction to tidymodels
 
 -   01: [Introduction](slides/01-introduction.html)
 -   02: [Your data budget](slides/02-data-budget.html)
 -   03: [What makes a model?](slides/03-what-makes-a-model.html)
 -   04: [Evaluating models](slides/04-evaluating-models.html)
 
-### Day Two
+### Advanced tidymodels
+
+-   01: [Tuning hyperparameters](slides/advanced-01-tuning-hyperparameters.html)
 
--   05: [Feature engineering](slides/05-feature-engineering.html)
--   06: [Tuning hyperparameters](slides/06-tuning-hyperparameters.html)
--   07: [Transit Case Study](slides/07-transit-case-study.qmd)
--   08: [Wrapping up](slides/08-wrapping-up.html)
 
 There's also a page for [slide annotations](slides/annotations.html); these are extra notes for selected slides.
```

Comment:
I'm going to wait until I've got good drafts of the advanced slides before updating the annotations.